Maven.gitignore

```diff
@@ -7,3 +7,6 @@ release.properties
 dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
+
+# Exclude maven wrapper
+!/.mvn/wrapper/maven-wrapper.jar
```

Comment:
I'm not quite sure about this rule - it isn't necessary unless a previous rule is hiding the `.mvn/wrapper/` folder.

Am I understanding the intent here?